model/project_aliases.go

```diff
@@ -23,31 +26,52 @@ const (
 //
 // For example, a user can specify that alias with the CLI tool so that a project
 // admin can define a set of default builders for patch builds. Pull request
-// testing uses a special alias, "github_pull_request" to determine the default
+// testing uses a special alias, "__github" to determine the default
 // variants and tasks to run in a patch build.
 //
 // An alias can be specified multiple times. The resulting variant/task
 // combinations are the union of the aliases. For example, a user might set the
 // following:
 //
 // ALIAS                  VARIANTS          TASKS
-// github_pull_request    .*linux.*         .*test.*
-// github_pull_request    ^ubuntu1604.*$    ^comppile.*$
+// __github               .*linux.*         .*test.*
+// __github               ^ubuntu1604.*$    ^compile.*$
 //
 // This will cause a GitHub pull request to create and finalize a patch which runs
 // all tasks containing the string “test” on all variants containing the string
 // “linux”; and to run all tasks beginning with the string “compile” to run on all
 // variants beginning with the string “ubuntu1604”.
 type ProjectAlias struct {
-	ID        bson.ObjectId `bson:"_id"`
-	ProjectID string        `bson:"project_id"`
-	Alias     string        `bson:"alias"`
-	Variants  string        `bson:"variants"`
-	Tasks     string        `bson:"tasks"`
+	ID        bson.ObjectId `bson:"_id" json:"_id"`
+	ProjectID string        `bson:"project_id" json:"project_id"`
+	Alias     string        `bson:"alias" json:"alias"`
+	Variant   string        `bson:"variant" json:"variant"`
+	Task      string        `bson:"task,omitempty" json:"task"`
+	Tags      []string      `bson:"tags,omitempty" json:"tags"`
 }
 
-// FindProjectAliases finds aliases with a given name for a project.
-func FindProjectAliases(projectID, alias string) ([]ProjectAlias, error) {
+// FindAliasesForProject fetches all aliases for a given project
+func FindAliasesForProject(projectID string) ([]ProjectAlias, error) {
+	out := []ProjectAlias{}
+	err := db.FindAll(ProjectAliasCollection,
+		bson.M{
+			projectIDKey: projectID,
+		},
+		db.NoProjection,
```

Comment:
I think you can leave these options blank rather than explicitly having them noop

Reply:
done